modules/salt/pages/large-scale.adoc

```diff
@@ -174,3 +174,15 @@ If significant non-transient swap usage is detected, you can increase the availa
 
 You can also consider tuning the Server to consume less memory.
 For more information on tuning, see xref:salt:large-scale-tuning.adoc[].
+
+=== AES Key rotation
+
+Communication from the Salt Master to all Salt minions is encrypted with a single AES key. This key is rotated whenever:
+ - the salt-master process is restarted, or
+ - any Salt minion key is deleted (eg. a client is deleted from {productname})
+
+Upon rotation, all minions will have to re-authenticate to the Master next time they receive a message, and if the number of minions is high (several thousands) then this is expected to cause noticeable CPU load on the {productname} Server.
+
+If the load caused by massive re-authentication is excessive, it is recommended to delete keys in batches, possibly in off-peak hours, not to overload the {productname} Server.
```

Comment:
```suggestion
If the CPU load is excessive, we recommend that you delete keys in batches, and in off-peak hours if possible, to avoid overloading the server.
```